### DISCUSSION TITLE
Disabling all Buttons and the Prefix of "Botname/Charactername:" before message

### DISCUSSION THREAD

**Discussion by ExperTnot** [innightwolfsleep/llm_telegram_bot#115] [Q&A]
Hi!
Im sorry if this is already possible and i just missed it.

I am currently using persona mode, which fits best while still being chat mode.
As said in the title, I want to ask for a mode where it seems like its not a bot and just a normal message.

Thanks you!

**Comment by innightwolfsleep** [comment]
Disabling buttons possible for all characters - change button visibility in telegram_user_rules.json. 
Disabling prefixes is default behaviour in notebook mode.

**Reply by ExperTnot** [reply]
I managed to remove the buttons :)

I tried using notebook mode.
when i use notebook the replies are different from chat and persona mode. not like a conversation anymore.
maybe im doing something wrong?

Or maybe I can somehow also easily remove the prefix like the buttons?

**Reply by innightwolfsleep** [reply]
> Or maybe I can somehow also easily remove the prefix like the buttons?

I got it) You mean, remove prefix from message, not from context (like notebook mode do).

Unfortunately, here is no respective mode. There is needed changes in code. Adding a text split/join in `prepare_text` method, for example.

**Reply by ExperTnot** [reply]
ok man thank anyways. :) 

should i mark as answer? or wait if you want to implement
